src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/ChangeApprenticeship.feature

```diff
@@ -0,0 +1,11 @@
+﻿@database
+@api
+
+Feature: ChangeApprenticeship
+
+Scenario: Update an apprenticeship with a new commitment statement
+	Given we have an existing apprenticeship
+	And we have an update apprenticeship request
+	When the update is posted
+	Then the result should return accepted
+	And the registration exists in database
```

Comment:
You're right, that and statement is not valid because the wording is wrong.  It should actually be `And the new commitment statement exists in database`.  I've updated it, and also added a new scenario for updates to apprenticeships we don't know about.